autoload/neomake/makers/ft/php.vim

```diff
@@ -35,8 +35,16 @@ function! neomake#makers#ft#php#phpcs() abort
 endfunction
 
 function! neomake#makers#ft#php#phpmd() abort
+    if exists('b:neomake_php_phpmd_args_standard')
+        let l:standard = b:neomake_php_phpmd_args_standard
+    elseif exists('g:neomake_php_phpmd_args_standard')
+        let l:standard = g:neomake_php_phpmd_args_standard
+    else
+        let l:standard =  'codesize,design,unusedcode,naming'
+    endif
+
     return {
-        \ 'args': ['%:p', 'text', 'codesize,design,unusedcode,naming'],
+        \ 'args': ['%:p', 'text', l:standard],
         \ 'errorformat': '%W%f:%l%\s%\s%#%m'
         \ }
 endfunction
```

Comment:
I do not  like the `l:` prefix (but can amend it).
It would be good to have some example output though to cover those cases.
Also `:let g:neomake_php_phpmd_args = neomake#makers#ft#php#phpmd().args + ['foo']` etc might work already?

Reply:
Thank you.  It is fixable by b:eomake_php_phpmd_args.  I will close this PR.